rebar.config

```diff
@@ -1,9 +1,11 @@
 % -*- mode: erlang -*-
 {erl_opts, [debug_info,
+            nowarn_export_all,
             {platform_define, "^R15", 'gen_tcp_r15b_workaround'},
             {platform_define, "^(R14|R15|R16B-)", 'crypto_compatibility'},
             {platform_define, "^(R14|R15|R16B|17)", 'rand_mod_unavailable'},
-            {platform_define, "^(R14|R15|R16)", 'map_unavailable'}]}.
+            {platform_define, "^(R14|R15|R16)", 'map_unavailable'},
+            {platform_define, "^2", 'crypto_rand_uniform_unavailable'}]}.
```

Comment:
This seems fragile as it will break if the version number ever goes beyond 2. Would be better to switch on old Erlang versions and use the rand module anywhere else.